geomesa-archetypes/geomesa-archetypes-gs-plugin/pom.xml

```diff
@@ -174,6 +174,23 @@
         </dependencies>
     </dependencyManagement>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-process-wps_${scala.binary.version}</artifactId>
```

Comment:
Where does this inclusion add the jar?  Is it part of the gs-plugin tar.gz?

Reply:
right, it makes it compile scope and then it gets picked up by the dependencies section of the maven assembly plugin